### DISCUSSION TITLE
shutterbutton seems only to work if.

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#17018] [Q&A] [answered]
Hallo . a short question.
shutterbutton seems only to work if the Switch/button (external) is defined in template section als being a button.. I am right?
I tried to define it as being a "switch" the command "shutterbutton1 xyz..." has no funktion @ all.. it doesn't matter wich "switchmode" is defined .. OR can someone tell how to configurate a "switch" (switchmode1/2 ???) that "shutterbutton" work??
Thanks and Regards
Mark-Muc

**Comment by sfromis** [comment]
Not using that feature, but https://tasmota.github.io/docs/Blinds-and-Shutters/ does talk about using Switch pins and SwitchMode. I know no more.....

**Reply by Mark-Muc** [reply]
thx sfromis . i already read this .. but no hint on the config of a "switch or button"
and my experience showed me that .. that's why I am asking!!
Mark-Muc

**Reply by Jason2866** [reply]
Define a switch. A switch can be hardware wise a momentary switch (=button).

**Reply by sfromis** [reply]
The docs page explicitly has phrases like
> If manual operation switches (Switch<x> or Button<x> pairs) are ...
> When using a switch for manual operation Switch<x> pairs should usually be set to SwitchMode<x> 2 (inverse follow) for ...
> using switches or buttons in the module configuration to ...

**Reply by Mark-Muc** [reply]
yep .. thats right .. i fond this .. i am talking about the command: SHUTTERBUTTON

**Comment by Mark-Muc** [comment, marked as answer]
Thx for helping out .. but I have to admit, that I am looking for an answer about the command 
`SHUTTERBUTTON `if `SO80 1` (shuttermode) ist activated.
https://tasmota.github.io/docs/Blinds-and-Shutters/#button-control
and i found:
`This assigns a Tasmota button <button> to control your shutter <x> having functionality <func>. The Tasmota button <button> must already be configured in the module configuration. You can assign multiple buttons to a single shutter. Any button can only control one shutter (beside the <mqtt> broadcast feature, see description below). Any press of the button while the shutter is moving will immediately stop the shutter.`
The answer is: 
**_The Tasmota button <button> must already be configured in the module configuration_**
Thx to all.. 
Regards Mark-Muc
PS. u can close this thread!
